scala3doc/src/dotty/renderers/DotDiagramBuilder.scala

```diff
@@ -22,8 +22,8 @@ object DotDiagramBuilder:
       s"""node${id} [id=node${id}, label="${getHtmlLabel(vertex, renderer)}", style="${getStyle(vertex)}"];\n"""
     }.mkString
 
-    val edges = diagram.edges.map { (from, to) =>
-      s"""node${vWithId(from)} -> node${vWithId(to)};\n"""
+    val edges = diagram.edges.map { case Edge(from, to, description) =>
+      s"""node${vWithId(from)} -> node${vWithId(to)} [label="${description}"];\n"""
```

Comment:
can we only add `labels` if are non-empty? I do not want to increase HTML size more then we  really need